recipes-kernel/linux-yocto/linux-yocto/0001-iio-tmp006-Set-correct-iio-name.patch

```diff
@@ -0,0 +1,33 @@
+From a1dbe23dac5f38c92714f69d0cac749c5e954aa7 Mon Sep 17 00:00:00 2001
+From: Yong Li <yong.b.li@intel.com>
+Date: Fri, 22 Apr 2016 11:28:24 +0800
+Subject: [PATCH] iio: tmp006: Set correct iio name
+
+When load the driver using the below command:
+echo tmp006 0x40 > /sys/bus/i2c/devices/i2c-0/new_device
+
+In sysfs, the i2c name is tmp006, however the iio name is 0-0040,
+they are inconsistent. With this patch,
+the iio name will be the same as the i2c device name
```

Comment:
can't say much, but maybe this is not right. If tmp006 is the module and you name the IIO after that, it means you can only have one IIO per driver, say you have one at 0x40 and another at 0x60.